packages/matter.js/src/cluster/server/CommandServer.ts

```diff
@@ -14,6 +14,32 @@ import { Endpoint } from "../../device/Endpoint.js";
 
 const logger = Logger.get("CommandServer");
 
+/**
+ * This error type allows a command handler to communicate the appropriate
+ * status code to CommandServer by throwing an exception.
+ */
+export class CommandError extends Error {
```

Comment:
Please check MatterError class and especially StatusResponseError which is kind of what you do here.  If you do not want to use them pleaser inherit from that

Reply:
I'll use StatusResponseError, looks like it's the same.